partiql-lang/build.gradle.kts

```diff
@@ -36,11 +36,10 @@ dependencies {
     api(project(":partiql-parser"))
     api(project(":partiql-spi"))
     api(project(":partiql-types"))
+    api(project(":partiql-plan"))
     api(Deps.ionElement)
     api(Deps.ionJava)
     api(Deps.ionSchema)
-    // libs are included in partiql-lang-kotlin JAR, but are not published independently yet.
-    libs(project(":partiql-plan"))
```

Comment:
Further up in this file, `libs` is defined. We can probably remove its declaration here for `v0.13.1`. Unsure whether we need to remove for all others, but as there will be development from `main`, this should be removed.

Reply:
I removed `libs` and its usages. Testing with local maven publishing still worked. Deleted `libs` in the other release branches.